src/sentry/api/endpoints/organization_events_spans_performance.py

```diff
@@ -257,17 +265,29 @@ def query_suspect_span_groups(
         functions_acl=["array_join", "sumArray", "percentileArray", "maxArray"],
     )
 
+    extra_conditions = []
+
     if span_ops:
-        builder.add_conditions(
-            [
-                Condition(
-                    builder.resolve_function("array_join(spans_op)"),
-                    Op.IN,
-                    Function("tuple", span_ops),
-                ),
-            ]
+        extra_conditions.append(
```

Comment:
Is there a reason you're collecting into `extra_conditions` first? You can just do `add_conditions([...])` twice, no?

Reply:
No specific reason